pkg/proc/registers.go

```diff
@@ -33,6 +34,9 @@ type Register struct {
 	Value string
 }
 
+type SavedRegisters interface {
```

Comment:
Instead of defining a new type can we just use the existing `proc.Registers` interface?

Reply:
Ah got it, I like that solution then.